### PR TITLE
chore(docs): initial documentation on hyperparameter setup

### PR DESCRIPTION
**Pull request by grantdfoster**
- adds initial documentation for hyperparameter setup for #95

### CODE REVIEW

**Comment by juanmanso** [comment]
> Can someone review - this PR today please?
> 
> @juanmanso @hide-on-bush-x @mudler @obasilakis

I'd say we stick to what we were already doing which is Reviewers + one line message to the channel. Because with all the notifications on-going, this is likely to slip.

Until we have some order in our GitHub, I'd say we do that

File: docs/internal/hyperparameters.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+btcli sudo set --param weights_rate_limit --value 5 --netuid 1 --subtensor.chain_endpoint ws://54.205.45.3:9945
+```
+
+## Testnet Setup
+
+- coming soon
+
+## Mainnet Setup
+
+- coming soon
```
Comment (juanmanso) [inline docs/internal/hyperparameters.md:29]: nitpick: wouldn't mention this if it's not ready

**Review by juanmanso — CHANGES_REQUESTED** [review]
Shouldn't this live as an internal docs? Because this is a blockchain type of configuration and not miner/validator specific, thus it is not a matter for the user that would come to this repo to run stuff, isn't it?

File: docs/internal/hyperparameters.md (hunk 1 above)
Comment (hide-on-bush-x) [reply docs/internal/hyperparameters.md:29]: +1

**Review by hide-on-bush-x — APPROVED** [review]

**Comment by grantdfoster** [comment]
closing this pull request, edits have been captured and ported to https://github.com/masa-finance/masa-bittensor-tools/pull/12

**Pull request closed without merging**